.github/workflows/dmd-cxx.yml

```
@@ -0,0 +1,70 @@
+# Github action to test C++ implementation of DMD
+#
+# Based-on:
+#  - Github C/C++ with Make
+#  - DMD C++ interop tests
+name: DMD C++ (phobos)
+
+on:
+  pull_request:
+    branches: [ dmd-cxx ]
+  push:
+    branches: [ dmd-cxx ]
+
+jobs:
+  main:
+    name: Run
+    strategy:
+      matrix:
+        os: [ ubuntu-16.04 ]
+        target: [ clang, g++ ]
+
+    # We're using the latest available images at the time of this commit.
```

```suggestion
    # We're using the oldest available images at the time of this commit.
```

? Otherwise you'd be on 18.04 or 20.04